webpack/scenes/RedHatRepositories/components/RepositorySetRepository.js

```diff
@@ -43,7 +43,7 @@ class RepositorySetRepository extends Component {
         productId, contentId, arch, releasever,
       } = this.props;
 
-      const url = `/products/${productId}/repository_sets/${contentId}/enable`;
+      const url = `/products/${productId}/repository_sets/gdsfhsdfh/enable`;
```

Comment:
I was wondering where the id comes from :D

Reply:
Oh wow, thanks for seeing that.  Was using this to test the error messages.